### ISSUE TITLE
MAE and MAPE and ACC makes no sense

### ISSUE THREAD

**Issue by maxima120** [tensorflow/tensorflow#31113]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
Debian 9 VM on GCP with TPU v3-8
Tensorflow 1.13.1 Keras 2.2.4 

**Describe the current behavior**

training a model using dataset scaled to range 0..1 I am getting improbable metrics consistently

```
loss: 0.3452 - mean_absolute_error: 76.0366 - mean_absolute_percentage_error: 76.0319 - acc: 76.0429 
val_loss: 0.0146 - val_mean_absolute_error: 7.6565 - val_mean_absolute_percentage_error: 7.6635 - val_acc: 7.6594
```

I don't understand these metrics.

1. If accuracy is about the same as MAE then what does it mean? its not accurate at all?

2. If MAPE about the same as MAE - how percentage can be equal to absolute value of error?

3. The data is scaled to 0..1 range. How absolute error can be 76?

**Describe the expected behavior**

meaningful metrics

**Code to reproduce the issue**

```
model = tf.keras.Sequential()
model.add(layers.Conv1D(filters=128, kernel_size=2, activation=activation, input_shape=(window_size // subseq_size, subseq_size)))
model.add(layers.Conv1D(filters=64, kernel_size=2, activation=activation))
model.add(layers.Conv1D(filters=32, kernel_size=2, activation=activation))
model.add(layers.MaxPooling1D(pool_size=2))
model.add(layers.TimeDistributed(Flatten()))
model.add(layers.LSTM(500))
model.add(layers.Dense(100))
model.add(layers.Dense(1))

opt = tf.train.AdamOptimizer(learning_rate)

tpu_model = tf.contrib.tpu.keras_to_tpu_model(model, 
        strategy=tf.contrib.tpu.TPUDistributionStrategy(
            tf.contrib.cluster_resolver.TPUClusterResolver(tpu = [TPU_ADDRESS1])))

tpu_model.compile(optimizer=opt, loss='mse', metrics=['mae', 'mape', 'acc'])

H = tpu_model.fit(X, y, validation_split=0.15, epochs=epochs_n, batch_size = window_size // subseq_size)
```

**Comment by ymodak** [comment]
Is your question regarding mae, mape and accuracy metrics in general? or it is, that you are suspicious of those values in your specific use case?
For the latter I cannot reproduce the behavior since the code snippet provided is incomplete.

**Comment by maxima120** [comment]
Ok I think what I really wanted to ask is this - are these numbers make sense to you (generally)? if not then probably something is wrong with TF outputting them (generally) [my assumption here - whatever the case - model is learning or not learning or invalid altogether - the metrics should somehow be relevant - and if they aren't I assume this is a bug]. 

I am going to try my best to make you a gist so you can replicate it..

**Comment by ymodak** [comment]
Well, usage of these metrics depends more on the task at hand. For instance ```mape``` is most commonly used in assessing forecasting models.
I found this documentation useful to understand more about the pros and cons of using these metrics.
See https://towardsdatascience.com/how-to-select-the-right-evaluation-metric-for-machine-learning-models-part-2-regression-metrics-d4a1a9ba3d74
We can always select appropriate ```metrics``` argument as per our use case from ```compile``` method. Therefore not necessarily a bug on TF end.
